src/drivers/nuopc/mediator/med.F90

```diff
@@ -475,18 +475,13 @@ subroutine InitializeP0(gcomp, importState, exportState, clock, rc)
 
     write(msgString,'(A,i6)') trim(subname)//' dbug_flag = ',dbug_flag
     call ESMF_LogWrite(trim(msgString), ESMF_LOGMSG_INFO, rc=dbrc)
-
-    if (dbug_flag > 5) then
-      call ESMF_LogWrite(trim(subname)//": called", ESMF_LOGMSG_INFO, rc=dbrc)
-    endif
+    call ESMF_LogWrite(trim(subname)//": called", ESMF_LOGMSG_INFO, rc=dbrc)
```

Comment:
why remove all the dbug_flags?   We need to agree on a way to do this that allows runtime control of these messages and implement it consistently across cime.

Reply:
We talked about this a while ago - and we agreed that the levels should go away and we should just get debug output in the PET files in debug mode.